tests/general/KeywordBearTest.py

```diff
@@ -84,6 +84,31 @@ def test_keyword_not_in_comment(self):
                           dependency_results=dep_results) as result:
             self.assertEqual(len(result[0].diffs), 0)
 
+    def test_keyword_in_multiple_comments(self):
+        text = ['# todo 1\n', '# todo 2']
+        comments = [SourceRange.from_values('F', 1, 1, 2, 8)]
+        dep_results = {
+            'AnnotationBear': [
+                self.annotation_bear_result_type({'comments': comments})
+            ]
+        }
+
+        with execute_bear(self.uut, 'F', text,
```

Comment:
Please use named keywords here. Refer line 224 of this file. Also please edit the lines in other test cases where named keywords are not used.